docker-compose.yml

```diff
@@ -115,6 +147,34 @@ services:
       - ./src/blocks:/blocks
       - blocks-node-modules:/blocks/node_modules
 
+  selenium:
+    image: ${SELENIUM_IMAGE:-selenium/standalone-chrome:4.8.3}
+    # image: ${SELENIUM_IMAGE:-seleniarm/standalone-chromium:4.8.3} # use this for ARM64
```

Comment:
I think this will be nicer instead of commented `image:`
```
# For ARM64 set `SELENIUM_IMAGE=seleniarm/standalone-chromium:4.0.0-20211213` in your `.env` file
```